main.py

```diff
@@ -8,6 +8,9 @@
 from selenium import webdriver
 from selenium.webdriver.support.ui import Select
 
+from webdriver_manager.chrome import ChromeDriverManager
+os.environ['WDM_LOG_LEVEL'] = '0'
```

Comment:
This should by passed through the constructor instead:
```python
ChromeDriverManager(log_level=0)
```

https://github.com/SergeyPirogov/webdriver_manager#wdm_log_level